src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxSerializer.cs

```diff
@@ -167,6 +172,34 @@ private void WriteNode(SyntaxNode node)
                 }
             }
 
+            private void WriteRazorDirective(RazorDirectiveSyntax node)
```

Comment:
q: why can't tests just use this?

Reply:
Just following what we do for IR nodes. We have two different serializers; one in source used for debugging and one in test used for baselining.

Reply:
kk